src/ui_elements/BetterLineEdit.gd

```diff
@@ -21,7 +21,7 @@ func _ready() -> void:
 	focus_entered.connect(_on_focus_entered)
 	focus_exited.connect(_on_focus_exited)
 	mouse_exited.connect(_on_mouse_exited)
-	text_submitted.connect(release_focus.unbind(1))
+	text_submitted.connect(func(): if not Input.is_action_just_pressed("ui_focus_next"): release_focus.unbind(1))
```

Comment:
I'd prefer to do this in its own method, instead of a lambda. Also, this should probably include the ui_focus_previous action too?